Automattic/pocket-casts-android
# Custom Playback Settings - Apply local/ global settings

## Description
This applies local/ global settings to the current playing episode using the new segemented bar control.

** Ignore local files for now

Part of #3042

## Testing Instructions
**Test Global Effects Settings**

1. Play an episode and open the Effects view
2. Make sure the `All Podcasts` is selected
3. Apply some changes
4. Verify that selecting `This podcast` updates the view by changing the settings
5. Verify that tapping back to `All podcasts` the previous settings are applied
6. Verify the global settings are applied to all podcasts episode and that the selected tab is always `All podcasts`
7. Navigate to the podcast page settings and verify that the local effects settings switch is off

**Test Local Effects Settings**

1. Play an episode and open the Effects view
2. Make sure the `All Podcasts` is selected
3. Select `This podcast` tab
4. Apply some changes
5. Verify that selecting `All podcasts` updates the view by changing the settings
6. Verify that tapping back to `This podcast` the previous settings are applied
7. Verify that the settings are applied to this podcast only 
8. Verify that when opening the Effects view in this podcast, the tab selected is always `This podcast`
9. Navigate to the podcast page settings and verify that the local effects settings switch is on (if you haven't subscribed to this podcast, make sure to subscribe to it)

## Screenshots or Screencast 

https://github.com/user-attachments/assets/72a2c883-62b9-4e58-a478-04e89235f055


## Checklist
 
#### I have tested any UI changes...
